### ISSUE TITLE
Load profile from file

### ISSUE THREAD

**Issue by FabianKramm** [devspace-cloud/devspace#1200]
**Is your feature request related to a problem?**  
The devspace.yaml can get very large and parts of it could be outsourced into other files. 

**Which solution do you suggest?**  
Add a new config option `profiles.*.from` that could look like this:

```yaml
version: v1beta9
profiles:
- name: production
  from:
    # Multiple options here
    file: path/to/other/devspace.yaml # Also an url would be possible
    profile: test
    # Or from git
    git: https://github.com/my-database-server
    tag: v3.0.1
    subPath: /configuration
    profile: test
```

**Comment by LukasGentele** [comment]
The git option should have branch, hash etc ideally with the same naming as in dependencies.

What happens if parent is defines for the profile? There are 2 cases: parent in this devspace.yaml and parent in the remote devspace.yaml. are both possible? Which parent would be used in case 1?

**Comment by FabianKramm** [comment]
Implemented with v5.2.0
